.travis.yml

```diff
@@ -0,0 +1,56 @@
+# Used old infrastructure, needed for integration tests:
+# http://docs.travis-ci.com/user/workers/standard-infrastructure/
+sudo: required
+language: python
+python:
+  - 2.7
+
+branches:
+  only:
+    - master
+    - /^v[0-9]+\.[0-9]+$/
+
+env:
+  global:
+    - CACHE_NAME=JOB1
+  matrix:
+    - TASK=ci-unit NODE_INDEX=0 NODE_TOTAL=2
+    - TASK=ci-unit NODE_INDEX=1 NODE_TOTAL=2
```

Comment:
I tried splitting Unit tests into 3 chunks, - but that fails because there are some tests which rely on the state of each other.

Integration tests are good - `4-5mins`, - we don't need to split them.